sycl/test/basic_tests/boolean.cpp

```diff
@@ -1,7 +1,8 @@
 // TODO: Enable compilation w/o -fno-sycl-early-optimizations option.
 // See https://github.com/intel/llvm/issues/2264 for more details.
+// XFAIL: gpu && (level_zero || opencl)
```

Comment:
Is there any plan to fix these `XFAIL`'s ?

Reply:
Yes. It's tracked here: https://github.com/intel/llvm/issues/2264.